### PR TITLE
Upgrade cql-exec-vsac

### PR DESCRIPTION
**Pull request by cmoesel**
The primary purpose of this PR was to upgrade cql-exec-vsac so we can use VSAC URLs as value set identifiers, but I did a few other extra things along the way:
* added a build.gradle so we can do cql-to-elm via `./gradlew cql2elm`
* fixed recent foot exam patients to always have a _recent_ foot exam
* upgraded eslint to resolve a major vulnerability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,19 @@ able to run the same example without credentials:
...
+
+The CQL execution framework uses the ELM JSON representations of the CQL logic.  After updating the example CQL in
+this project, you will need to translate it to ELM JSON for execution to pick up the changes. The CQL-to-ELM
+translator requires that [Java](https://adoptopenjdk.net/) be installed. This has been tested using OpenJDK 8, but
+later versions may also work (your mileage my vary).
+
+Once Java has been installed, run the CQL-to-ELM translation via the following command:
+
+```bash
+$ ./gradlw cql2elm
```
Comment (kjmahalingam) [inline README.md]: `gradlew` is spelled wrong here, but if I correct the spelling it works!

**Comment by kjmahalingam** [comment]
```
/-------------------------------------------------------------------------------
| CQL Execution Examples
\-------------------------------------------------------------------------------

[1] Diabetic Foot Example (FHIR DSTU2)
[2] Diabetic Foot Example (FHIR STU3)
[3] Diabetic Foot Example (FHIR R4)
[4] Patient Finder (FHIR DSTU2)
[5] Patient Finder (FHIR STU3)
[6] Patient Finder (FHIR R4)
[Q] Quit

Choose an example to run:

/-------------------------------------------------------------------------------
| Example:   Diabetic Foot Exam (dstu2)
| Usage:
|            node ./diabetic-foot-exam/dstu2.js vsacUser vsacPassword
|            node ./diabetic-foot-exam/dstu2.js
| VSAC User: kmahalingam
\-------------------------------------------------------------------------------

Error [ Error: Error downloading valueset: https://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.464.1003.103.12.1001
    at downloadValueSet.catch (/Users/kmahalingam/cql-exec-examples/node_modules/cql-exec-vsac/lib/download-vsac.js:35:18)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7) ]
```

I receive this error when trying to download a valueset running this example. Am I perhaps doing something wrong? I used what I believe to be my correct vsac username and password.

**Review by kjmahalingam — CHANGES_REQUESTED** [review]
A few minor comments but otherwise looks good.

**Comment by cmoesel** [comment]
Thanks, @kjmahalingam.  I've fixed the typo.

As for the value set download problem, I cannot reproduce that.  Did you run `yarn` to ensure that you got the latest dependency libraries?  That would make a difference.  If so, and it still doesn't work, this may be an issue w/ the MITRE network environment and SSL.  We can discuss on Slack if this might be the issue.

**Review by kjmahalingam — APPROVED** [review]
That did it, whoops! Good work.

**Pull request merged**